api/src/test/java/org/pmiops/workbench/api/UserControllerTest.java

```diff
@@ -129,15 +129,19 @@ public void testUnregistered() {
     when(fireCloudService.isUserMemberOfGroupWithCache(any(), any())).thenReturn(false);
     assertThrows(
         ForbiddenException.class,
-        () -> userController.user("Robinson", null, null, null).getBody());
+        () ->
```

Comment:
all of these changes are simply adding the Access Tier shortname to the new endpoint's signature